Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```
@@ -32,7 +32,7 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockClothesFilled
-  cost: 2480
+  cost: 2600
```

You need to mark your change with a `#Harmony` tag since you are editing upstream files.
```suggestion
  cost: 2600 # Harmony change: 2480 -> 2600
```